lib/rest-adapter.js

```diff
@@ -535,6 +535,7 @@ function RestClass(sharedClass) {
 
   this.name = sharedClass.name;
   this.routes = getRoutes(sharedClass);
+  this.sharedClass = sharedClass;
```

Comment:
`sharedClass` is already available as a non-enumerable read-only property - see L534 above.